cou/steps/execute.py

```diff
@@ -99,7 +116,7 @@ async def apply_plan(plan: UpgradeStep, interactive: bool) -> None:
     """
     result = ""
     while result.casefold() not in AVAILABLE_OPTIONS:
-        result = (await ainput(prompt(plan.description))).casefold() if interactive else "c"
+        result = await _user_input(plan, interactive)
```

Comment:
This implementation will have major changes when fixing #82 . Since your change is simply refactoring it to be more readable, it would be preferable to not touch this part in this PR to avoid merge conflict.

Reply:
I needed it to disable prompt, but that was in not working version, which was changed. This is no longer required, so I will remove it.